include/swift/AST/Decl.h

```diff
@@ -1893,13 +1892,13 @@ class IfConfigDecl : public Decl {
   ArrayRef<IfConfigDeclClause> Clauses;
   SourceLoc EndLoc;
   bool HadMissingEnd;
+  bool HasBeenResolved = false;
```

Comment:
Oh, the extra padding! It's not super important, but if you feel like packing these bools into the spare bits up in `Decl` in a follow-on PR, you'd have my micro-optimizing thanks